webcompat/static/js/lib/issues.js

```diff
@@ -489,6 +490,21 @@ issues.MainView = Backbone.View.extend({
 
   handleKeyShortcuts: function() {
     Mousetrap.bind("mod+enter", _.bind(this.addNewComment, this));
+  },
+
+  // Auto expanding feature for comments
+  // See Line 473 in bugform.js 
+  autoExpand: function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand", function() {
+      this.baseScrollHeight = this.scrollHeight;
+    })
+    .on("input.js-autoExpand", "textarea.js-autoExpand", function() {
+      var minRows = this.getAttribute("data-min-rows");
+      this.rows = minRows;
+      var rows = Math.ceil((this.scrollHeight - this.baseScrollHeight) / 10);
```

Comment:
It was to demonstrate the use of that number but I think that might get confusing later. We should use the same number.